dagql/objects.go

```diff
@@ -357,15 +357,15 @@ func NoopDone(res Typed, cached bool, rerr error) {}
 
 // Select calls the field on the instance specified by the selector
 func (r Instance[T]) Select(ctx context.Context, s *Server, sel Selector) (Typed, *call.ID, error) {
-	inputArgs, newID, doNotCache, err := r.preselect(ctx, s, sel)
+	inputArgs, newID, doNotCache, skipDedupe, err := r.preselect(ctx, s, sel)
```

Comment:
nit: lots of return args, maybe we should assemble them into a private struct?

Reply:
agreed, fixed that